cmd/flowserver/server.go

```diff
@@ -31,14 +31,14 @@ type FlowServer struct {
 }
 
 // NewFlowServer creates a new flow server instance
-func NewFlowServer(config *Config, logger *logrus.Logger) *FlowServer {
+func NewFlowServer(config *Config) *FlowServer {
 	r := chi.NewRouter()
 	r.Use(middleware.DefaultCompress)
 	r.Use(middleware.StripSlashes)
 	r.Use(middleware.RequestID)
 	r.Use(middleware.RealIP)
-	r.Use(traceErrors(logger))
-	r.Use(lg.RequestLogger(logger))
+	r.Use(traceErrors())
+	r.Use(lg.RequestLogger(log.StandardLogger()))
```

Comment:
Lets just nix lg.RequestLogger entirely and log requests ourselves after we handle them